src/config.ts

```
@@ -0,0 +1,7 @@
+import env from 'env-var';
+import process from 'node:process';
```

ESLint

[error] 'process' is defined but never used.

(@typescript-eslint/no-unused-vars)